source/etc/generate-backpressure-retryability-tests.py

```diff
@@ -0,0 +1,125 @@
+from collections import namedtuple
+from jinja2 import Template
+import os
+import sys
+
+Operation = namedtuple(
+    'Operation', ['operation_name', 'command_name', 'object', 'arguments'])
+
+CLIENT_BULK_WRITE_ARGUMENTS = '''models:
+          - insertOne:
+              namespace: retryable-writes-tests.coll
```

Comment:
Shouldn't this utilize a YAML reference so the names live all in once place (in the template)? See prior art in `clientBulkWrite` tests, as I don't believe they duplicated namespace lines like this.

`_yamlAnchors` would be the top-level key to use for this.